### PR TITLE
Footprint Tweaks

### LINKED ISSUE #11634: Walking on motor oil steps turn to blood steps
I downloaded the latest version for single player and walking on motor oil and other black icky stuff makes the shoemarks into gray blood marks. They look normal but when examined they're called "That's some blood."

### PR DESCRIPTION
**Pull request by RemieRichards**
- Footprints are now less common (10 steps per bloodpool, down from 20 steps), by making adding more footprints to the same footprint object take away blood from your shoes, which should also reduce the amount of "crossroad" patterns.
- One pool of blood will now only produce 1 trail of footprints
- Footprints do not occur from dried blood
- Footprints do not occur from tiny blood drops
- All footprints are now called "footprints", to fix Oil being referred to as blood
- Oil footprints are no longer valid sources of "blood" for Slaughter Demons
- Dried blood is no longer a valid source of blood for Slaughter Demons, only fresh (spawned in round) blood counts (Human or Xeno, as before)
- You can now tell what shoes footprints were made with, including a small image of the shoes (No unique sprites because that's not an easy thing to add to the system)
- New sprites for footprints by @WJohn, less swastika-y and appear at the same place in the tile as mulebot tracks, so they actually look like they came from the person's shoes.

Fixes #11634

### CODE REVIEW

**Comment by Chocobro** [comment]
oooo that last thing is real fancy
@WJohnston heels when

**Comment by Chocobro** [comment]
wow ok i'm too tired @WJohn

**Comment by Jalleo** [comment]
Image pls

File: code/game/objects/effects/decals/Cleanable/humans.dm [hunk 1]
```diff
@@ -101,38 +106,49 @@
...
+	return 1
 
 
 //BLOODY FOOTPRINTS
 /obj/effect/decal/cleanable/blood/footprints
+	name = "footprints"
 	icon = 'icons/effects/footprints.dmi'
 	icon_state = "nothingwhatsoever"
-	desc = "Follow the bloody brick road!"
+	desc = "where might they lead?"
```
Comment (Incoming5643) [inline code/game/objects/effects/decals/Cleanable/humans.dm]: Who's footprints are these?

Comment (Miauw62) [inline code/game/objects/effects/decals/Cleanable/humans.dm]: where should be capitalized reeeeee

**Comment by xxalpha** [comment]
REMOVE FOOTPRINTS
----------------

with a mop

**Comment by Dorsisdwarf** [comment]
@RemieRichards Fixes #11634

**Pull request merged**